### ISSUE TITLE
Looping audio source clicks audibly at loop point on windows

### ISSUE THREAD

**Issue by slime73** [love2d/love#1486]
**[Original report](https://bitbucket.org/rude/love/issue/1481) by Kira Boom (Bitbucket: [Kira Boom](https://bitbucket.org/Kira Boom), ).**

----------------------------------------

My looping audio source is making a clicking sound every time it loops, even though the beginning and end of the sound fade to 0. I've attached a minimal .love file that shows the issue.

The problem only happens on windows (both 32 and 64 bit), and only in Love2D builds 11.1 and later, 11.0 works fine. Also, if i overwrite OpenAL32.dll with the copy from Love2D 11.0, it fixes the issue in newer versions.

I've checked OSX, iOS, and Android builds, and they all seem fine.

So, in short:

- Love2D 11.0 on windows: works fine
- Love2D 11.1 on windows: clicks at loop point
- Love2D 11.2 on windows: clicks at loop point
- Love2D 11.1 or 11.2 on windows with OpenAL32.dll from 11.0: works fine
